subworkflows/local/prepare_intervals.nf

```diff
@@ -43,15 +45,20 @@ workflow PREPARE_INTERVALS {
         if (!params.intervals) {
 
             BUILD_INTERVALS(fasta_fai)
-            ch_intervals_combined = BUILD_INTERVALS.out.bed.map{it -> [[id:it.simpleName], it] }
+            ch_intervals_combined = BUILD_INTERVALS.out.bed
 
-            ch_intervals = CREATE_INTERVALS_BED(ch_intervals_combined)
+            CREATE_INTERVALS_BED(ch_intervals_combined)
```

Comment:
I have no idea why this is not working. I get an 
```
Invalid method invocation `call` with arguments: /work/af/b6d828ae6b237711b628339e0f9674/genome.fasta.bed (sun.nio.fs.UnixPath) on _closure14 type

```

However, if I remove the versions from the module and do this it works:

`CREATE_INTERVALS_BED(BUILD_INTERVALS(fasta_fai))`